src/modelmanager.cpp

```diff
@@ -239,7 +259,16 @@ void processPipelineConfig(rapidjson::Document& configJson, const rapidjson::Val
         }
         SPDLOG_DEBUG("Creating node: {} type: {} model_name: {} modelVersion: {}",
             nodeName, nodeKindStr, modelName, modelVersion.value_or(0));
-        info.emplace_back(std::move(NodeInfo{nodeKind, nodeName, modelName, modelVersion, nodeOutputNameAlias, demultiplyCount, gatherFromNode}));
+        info.emplace_back(std::move(NodeInfo{
```

Comment:
You don;t need the move here.

Probably you can just do:
info.emplace_back(nodeKind, node name, ....);

Reply:
Done